Lib/Magnet/HotKey.swift

```diff
@@ -9,7 +9,8 @@
 import Cocoa
 import Carbon
 
-public final class HotKey: Equatable {
+@objc(HotKey)
+public final class HotKey: NSObject {
```

Comment:
Please change `Equatable` of `NSObject` must override `isEqual(_:)`.
Like this https://github.com/Clipy/Magnet/blob/33a13816b27e20e39e580af1e001099190caa7e3/Lib/Magnet/KeyCombo.swift#L84-L90

Reply:
Changed to override isEqual with e5cf7f0